CONTRIBUTING.md

```diff
@@ -28,5 +28,25 @@ We use GitHub issues to track public bugs. Please ensure your description is cle
 * "Attractive"
 * Do not use the optional parameters of `setTimeout` and `setInterval`
 
+## Testing
+
+To run tests in the browser, run
+
+```
+npm run test:server
+```
+
+Open http://localhost:8081/test. Anytime you change test files, the tests will automatically be rerun, no refresh
+required! You can even click a specific section to only run those tests.
+
+To run tests in the CLI, run
+
+```
+npm run test
+```
+
+This will fail on some errors that npm run test:server won't, so it's recommended that you run this command after you've
+finished developing on tests with npm run test:server, if you chose that method.
```

Comment:
https://github.com/schrodinger/fixed-data-table-2/pull/167#discussion_r119945925: "Before I removed this, >npm run test failed most FDTRoot-test tests with TypeError: "callback" argument must be a function, whereas they all passed with >npm run test:server"

That's all I got. I spent time investigating last week but to no avail, unfortunately. All I can think of is that the former runs mocha-webpack, while the latter runs webpack-dev-server, so maybe there's a difference in mocha-webpack? Not sure. I'll look into it a bit more.